setup-system.sh

```diff
@@ -43,8 +43,16 @@ function install_sublime_3() {
     sudo dpkg -i /tmp/sublime-text_build-3065_amd64.deb
 }
 
+# Download and install atom as best we know how
+function install_atom() {
+    echo "~ Downloading atom deb package"
+    sudo wget -P /tmp https://atom-installer.github.com/v1.5.3/atom-amd64.deb
```

Comment:
Sure I was copying the sublime function. Would you like me to reflect these
changes for sublime too?

On Fri, 19 Feb 2016, 09:47 Robin Winslow <notifications@github.com> wrote:

> In setup-system.sh
> <https://github.com/nottrobin/system-setup/pull/3#discussion_r53440589>:
>
> > @@ -43,8 +43,16 @@ function install_sublime_3() {
> >      sudo dpkg -i /tmp/sublime-text_build-3065_amd64.deb
> >  }
> >
> > +# Download and install atom as best we know how
> > +function install_atom() {
> > +    echo "~ Downloading atom deb package"
> > +    sudo wget -P /tmp https://atom-installer.github.com/v1.5.3/atom-amd64.deb
>
> Could we make this wget -O /tmp/atom.deb https://atom.io/download/deb?
> That way it'll track the latest version. Also the wget part shoudn't need
> sudo - anyone can use the /tmp directory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nottrobin/system-setup/pull/3/files#r53440589>.
>